src/krbd.cc

```diff
@@ -764,3 +764,58 @@ int krbd_showmapped(struct krbd_ctx *ctx, Formatter *f)
 {
   return dump_images(ctx, f);
 }
+
+int krbd_is_image_mapped(struct krbd_ctx *ctx, const char *poolname, 
+                       const char *imgname, const char *snapname,
+                       std::ostringstream &mapped_info) {
+  struct udev_enumerate *enm;
+  struct udev_list_entry *l;
+  struct udev *udev = ctx->udev; 
+  const char *mapped_id, *mapped_pool, *mapped_image, *mapped_snap;
+  int r = 1;
```

Comment:
Thanks for your advice, I think you are right.
ret value r<0 means didn't get mapped info, r=0 means mapped. when r<=0, there should be a warning, while r >0, nothing need to be done.
I'll rename the function name.